openmls/src/framing/public_message.rs

```diff
@@ -291,11 +295,53 @@ impl<'a> ConfirmedTranscriptHashInput<'a> {
     }
 }
 
+#[cfg(test)]
```

Comment:
I think that {Confirmed,Interim}TranscriptHash{Input,} can be made very modular, and that we can remove all other `calculate_` methods everywhere. I can open an issue to do this in a later cleanup phase.

Reply:
Can you make use of these functions in the code already? Then we don't need to add the test cfg and would know we are testing the same thing.